src/client/api/pythonApi.ts

```diff
@@ -173,7 +173,12 @@ export class PythonExtensionChecker implements IPythonExtensionChecker {
         const yes = localize.Common.bannerLabelYes();
         const no = localize.Common.bannerLabelNo();
         sendTelemetryEvent(Telemetry.PythonExtensionNotInstalled, undefined, { action: 'displayed' });
-        const answer = await this.appShell.showErrorMessage(localize.DataScience.pythonExtensionRequired(), yes, no);
+        const answer = await this.appShell.showErrorMessage(
```

Comment:
Should this one be a warning too? Maybe not.

Reply:
Oh yes, you're right, will change to information.